.scalafmt.conf

```diff
@@ -0,0 +1,15 @@
+style = defaultWithAlign
+
+align.openParenCallSite = false
+align.openParenDefnSite = false
+align.tokens = [{code = "->"}, {code = "<-"}, {code = "=>", owner = "Case"}]
+continuationIndent.callSite = 2
+continuationIndent.defnSite = 2
+danglingParentheses = true
+indentOperator = spray
```

Comment:
Can this be akka to better match the project? I know spray and akka both lead to akka, so more of a nit.